docs/how_we_work/issue_review.md

```diff
@@ -0,0 +1,12 @@
+# Reviews Process for Issues
+
+As a team we will use labels for tracking which reviews are assign to whom and issue comments to tag our teammates when an issue is ready for their review.
+
+## Process
+
+- When an issue is ready for review, move to "In Review"
+- Add `____ Review` label for those who need to review and also comment, "@" tagging them on the issue stating that it's ready for their review
```

Comment:
Should we mention something about the default being 1 PM + 1 Eng review per issue?

Reply:
Good catch! Yes.

Reply:
Done when you're ready to re-review. This view doesn't show the change, but if you look at the file you can see it :)